src/main/java/it/pagopa/transactions/utils/PaymentGateway.java

```diff
@@ -0,0 +1,16 @@
+package it.pagopa.transactions.utils;
+
+/**
+ * Enum class to handle different payment gateways
+ */
+public enum PaymentGateway {
+    VPOS("VPOS"),
+    XPAY("XPAY"),
+    POSTEPAY("POSTEPAY");
+
+    public final String value;
+
+    PaymentGateway(String paymentGateway) {
+        this.value = paymentGateway;
+    }
+}
```

Comment:
Can you put this enumeration into commons and change paymentGateway event from String to PaymentGateway? 
In this way we can use the same enumeration also into event-dispatcher.
What do you think?